1.Exploratory_Data_Analysis/2.single_channel_umap_analysis.ipynb

```diff
@@ -0,0 +1,598 @@
+{
```

Comment:
<p>May also consider storing these in a dictionary</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature_data_analysis/pull/48/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>